frontend/packages/operator-lifecycle-manager/integration-tests-cypress/views/operator.view.ts

```diff
@@ -209,6 +209,20 @@ export const operator = {
     cy.byTestID('name-filter-input').clear();
     cy.byTestID('name-filter-input').type(name);
   },
+  modalElementShouldBeVisible: (testId, message) => {
```

Comment:
I didn't realize changing the check to 'is.visible' would cause this extra complexity. Let's keep it as 'exists' to make this test less fragile.

Reply:
> I didn't realize changing the check to 'is.visible' would cause this extra complexity. Let's keep it as 'exists' to make this test less fragile.

IMO, we should leave it as it is since it verifies that the element is visible to the user. I don’t think it will cause any issues like flakes for now, but we can keep an eye on it.  Also, the method looks clear to me and is abstracted from the actual test. We might need such tests for modals in other parts of the console later on.

Reply:
I'm specifically curious about the css styles we are manipulating. Forcing styles on the DOM doesn't feel advisable in an e2e test.